docs/docs/overview.md

```diff
@@ -6,10 +6,18 @@ description: Secretless Broker Documentation
 permalink: docs/overview.html
 ---
 
-The Secretless Broker is a connection broker which relieves client applications of the need
-to directly handle secrets to target services such as databases, web services, SSH
-connections, or any other TCP-based service.
+The Secretless Broker is a connection broker that relieves client applications of
+the need to directly handle secrets. When an application requires access to a
+Target Service such as a database, web service, SSH connection, or any other
+TCP-based service, rather than connect to the Target Service directly it can
```

Comment:
`or any other TCP-based service` isn't true - it's only for services we explicitly support.

Reply:
addressed by adding link to `reference` page on bottom.